src/lib/button/button.html

```diff
@@ -1,6 +1,7 @@
 <span class="mat-button-wrapper"><ng-content></ng-content></span>
 <div md-ripple *ngIf="!_isRippleDisabled()" class="mat-button-ripple"
-    [class.mat-button-ripple-round]="_isRoundButton"
-    [mdRippleTrigger]="_getHostElement()"></div>
+     [class.mat-button-ripple-round]="_isRoundButton || _isIconButton"
```

Comment:
But it still checks for the `icon` as well? [See here](https://github.com/angular/material2/pull/3208/files#diff-96337821f283af0f38c4fd8ed4c98065R108)